src/site/_includes/partials/footer.njk

```diff
@@ -0,0 +1,128 @@
+<footer class="w-footer">
+  <nav class="w-footer__linkboxes">
+    <ul class="w-footer__linkboxes-list">
+      <li class="w-footer__linkbox">
+        <h3 class="w-footer__linkbox-heading">Contribute</h3>
+        <ul class="w-footer__linkbox-list">
+          <li class="w-footer__linkbox-item">
+            <a href="https://github.com/GoogleChrome/web.dev/issues/new?assignees=&labels=bug&template=bug_report.md&title="
```

Comment:
`target="_blank" rel="noopener"` on all links?

Reply:
I think the devsite policy is not to use target=_blank.

> We recommend not using target="_blank" to force such links to open a new tab; developers know how to open new tabs if they want to, so we should respect their decisions and leave the choice to them. For more thoughts on use of target="_blank", read [Paul Ingraham's blog](http://www.paulingraham.com/target-blank-is-evil.html).

The linked post makes a decent argument so I'm inclined to keep them if that's ok.